docs/src/styles/docs/base.scss

```diff
@@ -303,6 +303,31 @@ h4:hover > a .icon-link {
   }
 }
 
+.expand-out {
+  // Only expand things for people without reduced motion preference
+  @media (prefers-reduced-motion: no-preference) {
+    &.shown {
+      animation: expand-out calc(2 * var(--amplify-time-long))
+        cubic-bezier(0.2, 0.6, 0.3, 1) both;
+    }
+  }
+}
+
+@keyframes expand-out {
+  0% {
+    letter-spacing: calc(-1 * var(--amplify-space-xs));
+    transform: translateZ(-500px);
```

Comment:
Arbitrary number based on manual testing, it works without this entirely, but the expansion is less smooth and less noticeable.

Reply:
Could you add a comment in here so a future developer knows this?

Reply:
Done!